articles/concepts/circuits.md

```diff
@@ -19,7 +19,7 @@ The circuit diagram for preparing this maximally entangled quantum state is:
 
```

Comment:
Honestly, looking through this page to remind myself of the context, I'm not actually sure that the circuits page really fits into the restructured docs at all? It might be best to drop the page entirely, perhaps. Thoughts, @KittyYeungQ?

Reply:
Because it was written for the teleportation example way back when? I think we still need a page to explain circuits though, for beginners to know about this widely used concept in quantum. We need to actually re-do the whole concepts section, hence the learning contents to be created. We can keep this before that happens.